core/src/main/resources/oracle/weblogic/deploy/messages/wlsdeploy_rb.properties

```diff
@@ -1262,6 +1267,8 @@ WLSDPLY-18024=Detected integer value {0} that could not be parsed to an integer
 WLSDPLY-18025=Detected float value {0} that could not be parsed to a floating point number so it will be set to 0: {1}
 WLSDPLY-18026=Detected number field with an empty value so it will be set to 0
 WLSDPLY-18027=Element {0} has an unknown value type {1} so its value will be set to None
+WLSDPLY-18028=Unable to parse the model YAML file because it contains a duplicate category entry {0}
+WLSDPLY-18029=Unable to parse the model JSON file because it contains a duplicate category entry {0}
```

Comment:
Seems like 18028 and 18029 could have been the same message, because you are going to tell the user what type of file it was in the next error message (your catch in each translator).
Unable to parse the model file because it contains a duplicate category entry {0}